iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/StaticResourcesProvider.java

```diff
@@ -0,0 +1,51 @@
+package eu.dnetlib.iis.wf.importer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+
+import com.google.common.collect.Lists;
+
+
+/**
+ * Encapsulates list of resources created from classpath locations.
+ * 
+ * @author mhorst
+ *
+ */
+public class StaticResourcesProvider {
+
+    /**
+     * List of resources to be provided.
+     */
+    private final List<String> resources;
+    
+    //------------------------ CONSTRUCTORS --------------------------
+    
+
+    /**
+     * @param resourcesLocations resources to be read classpath locations
+     */
+    public StaticResourcesProvider(String ...resourcesLocations) {
```

Comment:
And the main thing here: I don't think you need to do these io operations in constructor. In my view, you should have written a static method that reads the properties from the given classpaths, that's all. It's just an ordinary util class, isn't it?